typed-values/pom.xml

```
@@ -24,7 +24,13 @@
       </dependency>
     </dependencies>
   </dependencyManagement>
-
+  <properties>
+    <camunda.artifact>org.camunda.bpm.engine.variable</camunda.artifact>
+    <camunda.osgi.import.pkg>
+      !javax.activation,
+      *
+    </camunda.osgi.import.pkg>
```

I am wondering if we can use the `camunda.osgi.import.additional` element here as well to define `!javax.activation`?
```suggestion
    <camunda.osgi.import.additional>
      !javax.activation
    </camunda.osgi.import.additional>
```